governator-jersey/build.gradle

```diff
@@ -14,6 +14,10 @@ dependencies {
 
     compileOnly 'javax.servlet:javax.servlet-api:3.0.1'
 
+    implementation "javax.xml.bind:jaxb-api:2.3.1"
+    runtime "com.sun.xml.bind:jaxb-core:2.3.0.1"
+    runtime "com.sun.xml.bind:jaxb-impl:2.3.2"
```

Comment:
If Jaxb is not being used but it's annotation are loaded when a class is loaded, it would suffice to just add the jaxb-api.
Also, an alternative to the split package jaxb implementation is `jaxb-runtime` from glassfish.
https://www.jesperdj.com/2018/09/30/jaxb-on-java-9-10-11-and-beyond/